src/pages/billing/List.js

```diff
@@ -87,41 +255,64 @@ export const BillingList = ajaxCaller(class BillingList extends Component {
           }
         })
       ]),
-      h(PageBox, { style: { padding: '1.5rem', flex: 1 } }, [
+      h(PageBox, [
         div({ style: Style.cardList.toolbarContainer }, [
-          div({
-            style: {
-              ...Style.elements.sectionHeader,
-              textTransform: 'uppercase',
-              marginBottom: '1rem'
-            }
-          }, ['Billing Projects Management'])
+          div({ style: { ...Style.elements.sectionHeader, textTransform: 'uppercase' } }, ['Billing Projects Management']) //Billing Projects? Billing Management? Billing Project Management?
         ]),
         div({ style: Style.cardList.cardContainer }, [
+          h(NewBillingProjectCard, {
+            onClick: () => {
+              if (Auth.getAuthInstance().currentUser.get().hasGrantedScopes('https://www.googleapis.com/auth/cloud-billing')) {
+                this.setState({ creatingBillingProject: true })
+              } else {
+                const options = new window.gapi.auth2.SigninOptionsBuilder({ 'scope': 'https://www.googleapis.com/auth/cloud-billing' })
+                Auth.getAuthInstance().currentUser.get().grant(options).then(
+                  () => setTimeout(() => this.setState({ creatingBillingProject: true }), 250),
```

Comment:
This timeout is because Rawls was throwing an error after a user granted additional scopes. Apparently it needs a few milliseconds for the update to propagate through Rawls. Is there another way to accomplish this?